components/google_sheets/sources/new-updates-shared-drive/new-updates-shared-drive.mjs

```diff
@@ -0,0 +1,25 @@
+import httpBase from "../common/http-based/drive.mjs";
+import newUpdates from "../common/new-updates.mjs";
+
+export default {
+  ...httpBase,
+  ...newUpdates,
+  key: "google_sheets-new-updates-shared-drive",
+  type: "source",
+  name: "New Updates (Shared Drive, Instant)",
+  description: "Emit new event each time a row or cell is updated in a spreadsheet in a shared drive",
+  version: "0.0.1",
+  dedupe: "unique",
+  hooks: {
+    ...httpBase.hooks,
+    ...newUpdates.hooks,
+  },
```

Comment:
Since `newUpdates` doesn't have any hooks, and the hooks from `httpBase` will be included anyway, this hooks section can be eliminated.
```suggestion
```